zombienet/tests/0007-full_node_warp_sync.feature

```diff
@@ -7,6 +7,11 @@ bob: is up
 charlie: is up
 dave: is up
 eve: is up
+ferdie: is up
+george: is up
+george: pause
 alice: parachain 2000 is registered within 225 seconds
 dave: reports block height is at least 7 within 250 seconds
 eve: reports block height is at least 7 within 250 seconds
+george: resume
+george: reports block height is at least 7 within 250 seconds
```

Comment:
Maybe the time required to sync could be much less then time required for building blocks?

Reply:
We can reduce it but I am not even sure what the right numbers should be given I cannot run the tests locally